apps/web/src/api/auth.ts

```diff
@@ -44,6 +44,7 @@ async function registerUserFromGitHub(accessToken: string) {
     await User.query().findById(user.id).patch(userData);
   } else {
     user = await User.query().insertAndFetch(userData);
+    await Account.query().insert({ userId: user.id });
```

Comment:
And the organization? In synchronizer it is not done.

Reply:
First, there is no organization without account in the database. Moreover, I have check the code and I didn't find any line where an Organization is created without account. I think we already take care of creating account after Organization creation.